backtester/engine/grpcserver.go

```diff
@@ -210,6 +210,28 @@ func (s *GRPCServer) ExecuteStrategyFromFile(_ context.Context, request *btrpc.E
 		return nil, err
 	}
 
+	io64 := int64(request.IntervalOverride)
+	if io64 > 0 {
+		if io64 < gctkline.FifteenSecond.Duration().Nanoseconds() {
+			return nil, fmt.Errorf("%w %v", gctkline.ErrInvalidInterval, request.IntervalOverride)
+		}
+		cfg.DataSettings.Interval = gctkline.Interval(request.IntervalOverride)
+	}
+
+	if request.StartTimeOverride != nil && !request.StartTimeOverride.AsTime().IsZero() {
```

Comment:
request.StartTimeOverride.AsTime() calls `time.Unix` with zero values so this is always false even if nil.

Reply:
```go
func TestNil(t *testing.T) {
	t.Parallel()
	hello := timestamppb.New(time.Now())
	hello = nil
	if hello.AsTime().Unix() == 0 {
		t.Log("hi")
	}
}
```
Thanks! Didn't know!